pkg/scheme/core/v1/cri/docker.go

```diff
@@ -21,13 +21,20 @@ package cri
 import (
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"os"
 	"path/filepath"
 	"runtime"
+	"sort"
+	"strings"
 	"time"
 
+	"github.com/docker/docker/daemon/config"
```

Comment:
seems build error
![Uploading QQ20221021-150205.png…]()

Reply:
use map[string]interface unmarshall docker config, add update value of  map[insecure-registryies]